test/globals.js

```diff
@@ -43,4 +43,8 @@ testFixture.ARCHITECTURE = {
     Seq2Seq: '/4'
 };
 
+testFixture.LANGUAGE = {
```

Comment:
remove this (not necessary since the activeNode doesn't need to be `Language`)

Reply:
Removed